carbon-design-system/carbon-components-svelte
# Release workflow improvements

Introducing some semi-automatic release tooling:

- remove `husky`
- add linting to PR pipelines
- add release guide to contributing.md, and table of contents
- add `yarn release` semi-automatic script, leveraging the `standard-version` package
- Lint and fix neglected files. I assume their line endings were incorrect

How to test:

1. Check guide in CONTRIBUTING.md
2. Run `yarn release`. This won't create a commit nor tag; those steps are manual. Note the generated changelog, package.json version bump, and documentation version bump

Next steps: once we're comfortable with these tools, we'll set up continuous delivery.
